apps/web/app/(authenticated)/(app)/app/settings/user/update-user-name.tsx

```diff
@@ -64,10 +65,10 @@ export const UpdateUserName: React.FC = () => {
               });
               user.reload();
             })
-            .catch(() => {
+            .catch((error) => {
               toast({
                 title: "Error",
-                description: "Sorry there was an error updating your username",
+                description: (error as ClerkError).errors[0].longMessage,
```

Comment:
just in case that doesn't exist or it's a length 0 array, cause right now it would crash the entire site, which is even worse than a bad error message 

```suggestion
                description: (error as ClerkError).errors.at(0)?.longMessage ?? "Sorry there was an error updating your username",
```